src/v2/Components/Payment/SavedCreditCards.tsx

```diff
@@ -135,16 +135,22 @@ export class CreditCard extends React.Component<
       const creditCardEdge = creditCardOrErrorEdge.getLinkedRecord("creditCard")
       const creditCardId = creditCardEdge.getValue("id")
       const meStore = store.get(this.props.me.id)
-      const connection = ConnectionHandler.getConnection(
+      let connection = null
+      connection = ConnectionHandler.getConnection(
         meStore,
         "UserSettingsPayments_creditCards"
       )
+      if (!connection) {
+        connection = ConnectionHandler.getConnection(
+          meStore,
+          "PaymentSection_creditCards"
+        )
+      }
```

Comment:
This component is being used in 2 places so depending on the parent component the name of the relay connection would be different